setup.cfg

```diff
@@ -58,36 +58,35 @@ package_dir =
 install_requires =
     pip >= 22
     setuptools >= 60
-    wheel ~= 0.37
-    cookiecutter ~= 2.1
-    tomli ~= 2.0; python_version <= "3.10"
+    wheel >= 0.37
+    cookiecutter >= 2.1, < 3.0
+    tomli >= 2.0, < 3.0; python_version <= "3.10"
     importlib_metadata >= 4.4; python_version <= "3.9"
-    requests ~= 2.28
-    GitPython ~= 3.1
-    dmgbuild ~= 1.6; sys_platform == "darwin"
-    psutil ~= 5.9
-    rich ~= 12.6
-    platformdirs ~= 2.6
-    packaging ~= 22.0
-    tomli_w ~= 1.0
+    requests >= 2.28, < 3.0
+    GitPython >= 3.1, < 4.0
+    dmgbuild >= 1.6, < 2.0; sys_platform == "darwin"
+    psutil >= 5.9, < 6.0
+    rich >= 12.6, < 14.0
+    platformdirs >= 2.6, < 3.0
+    packaging >= 22.0
+    tomli_w >= 1.0, < 2.0
 
 [options.extras_require]
 dev =
-    pre-commit~=2.20
-    setuptools_scm[toml] ~= 7.0
-    tox ~= 4.0
-    coverage[toml] ~= 7.0
+    pre-commit == 2.21.0
+    setuptools_scm[toml] == 7.0.5
+    tox == 4.0.1
 test =
-    pytest~=7.2
-    pytest-tldr~=0.2
-    pytest-cov~=4.0
+    pytest == 7.2.0
+    pytest-tldr == 0.2.5
+    coverage[toml] == 7.0.4
```

Comment:
A development environment should probably have all the test tools installed, so Toga will merge`[test]` into `[dev]`as soon as the setuptools-scm PR is merged (https://github.com/beeware/toga/pull/1675#pullrequestreview-1178871290). Would it make sense to do the same for Briefcase?

Reply:
Yeah - the distinction is pretty narrow, so I guess that makes sense, if only for simplicity.